packages/plugin-vue-jsx/src/index.ts

```diff
@@ -74,7 +74,7 @@ function vueJsxPlugin(options: Options = {}): Plugin {
     },
 
     async transform(code, id, opt) {
-      const ssr = typeof opt === 'boolean' ? opt : (opt && opt.ssr) === true
+      const ssr = opt?.ssr
```

Comment:
```suggestion
      const ssr = opt?.ssr === true
```

Perhaps we can have `=== true` here for consistency with the other changes too.